kube_downscaler/cmd.py

```diff
@@ -36,7 +38,7 @@ def get_parser():
         "--include-resources",
         type=check_include_resources,
         default="deployments",
-        help="Downscale resources of this kind as comma separated list. [deployments, statefulsets, stacks] (default: deployments)",
+        help="Downscale resources of this kind as comma separated list. [deployments, statefulsets, stacks, horizontalpodautoscalers] (default: deployments)",
```

Comment:
I guess it makes sense to build this string now from the `VALID_RESOURCES` constant.

Reply:
Done